tutorials/src/bin/basic-tutorial-4.rs

```diff
@@ -0,0 +1,138 @@
+extern crate gstreamer as gst;
+use std::io;
+use std::io::Write;
+use gst::prelude::*;
+use gst::MessageView;
+
+struct CustomData {
```

Comment:
I don't like the name, but that's how it is in C so whatever... :)

Reply:
@lovebug356 Can you also add a PR for tutorial 3 that adds this struct for more similarity with the C code?

Reply:
Ok, I will have a look at it.